script/build.go

```diff
@@ -139,8 +145,8 @@ func buildCommand(dir, buildos, buildarch string) error {
 
 	args := make([]string, 1, 6)
 	args[0] = "build"
-	if len(LdFlag) > 0 {
-		args = append(args, "-ldflags", LdFlag)
+	if len(LdFlags) > 0 {
+		args = append(args, "-ldflags", strings.Join(LdFlags, " "))
```

Comment:
Ahh, you're right, my bad - forgot that `-ldflags` was its own nested beast. I guess we just deal with any quoting issues manually if/when they occur then.